src/client/containers/Home/Home.js

```diff
@@ -13,8 +13,18 @@ export const Home = () => (
         </div>
         <h1>Treasure Hunt</h1>
       </div>
-      <SubmitButton ButtonTitle="Join Game" />
-      <SubmitButton ButtonTitle="Start Game" />
+      <SubmitButton
+        ButtonTitle="Join Game"
+        onClick={() => {
+          location.href = '../share-game-page';
+        }}
+      />
+      <SubmitButton
+        ButtonTitle="Start Game"
+        onClick={() => {
+          location.href = '../create-team-page';
+        }}
+      />
     </div>
     <Footer title="Test title" />
```

Comment:
Just curious, what this title="Test title" is referring too? The name is a bit confusing.

Reply:
It is referring to nothing, i don't know why i wrote it. I will delete it, good catch Lucha 👍